plugins/google-analytics-dashboard/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC } from 'react';
+import { TextField, MenuItem, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  root: {
+    '& .MuiTextField-root': {
```

Comment:
these are inside a card, right? maybe the card could decide the size of the contents? in that case you can set the `fullWidth` prop on the `TextField`s